### PR TITLE
Closes #187

### LINKED ISSUE #187: model class `Effloglog` cleanup
Please edit appropriately below if needed.

To do:

- [ ] Revise documentation
  - [ ] RStudio outline section dividers (use `# function or class name ----`)
  - [ ] Add lifecycle badge (as a default take "stable" for existing functions or classes)
  - [ ] Replace `##'` by `#'`
  - [ ] Add to `pkgdown.yml`
  - [ ] Remove file history and the timestamp in comments on top of file
- [ ] Add unit tests
  - [ ] For classes:
    - [ ] Test default constructor
    - [ ] For reference classes: test methods
  - [ ] For functions: as usual
- [ ] Refactor as needed
  - [ ] Code styling (indentation etc.) - install package `styler` and use corresponding add-in
  - [ ] Keep externally used names so that we don't break downstream code
  - [ ] Remove `keywords` tags
  - [ ] For functions or methods:
    - [ ] Add assertions for arguments using `checkmate` functions
    - [ ] If it is too long, break up into pieces (rule of thumb: maximum 50 lines of code)
    - [ ] Consider renaming internal variables etc. if needed
    - [ ] Avoid explicit use of `return`
  - [ ] For classes:
    - [ ] move function definitions outside if possible (e.g. move validation function as `validity_subjects()` to `Data-validity.R` and have corresponding test file `test-Data-validity.R`)
    - [ ] In `setClass` function, just use the `slots` instead of outdated `representation`
    - [ ] Remove things like `validObject(.Data())`

### PR DESCRIPTION
**Pull request by wwojciech**
No unit tests yet.

### CODE REVIEW

File: R/Model-class.R [hunk 1]
```diff
@@ -1938,239 +1936,227 @@ LogisticIndepBeta <- function(binDLE,
...
+  nobs_no_dlt <- length(eff_obsrv)
+  if (nobs_no_dlt > 0L) { # Observed data available.
+    # Set X, Y to observed data only.
+    X <- model.matrix(fit_eff)[-seq_along(eff), ]
+    Y <- eff_obsrv
+
+    fit_eff0 <- lm(eff ~ log(log(eff_dose))) # Pseudo only.
+    X0 <- model.matrix(fit_eff0)
+    mu0 <- coef(fit_eff0)
+    Q0 <- t(X0) %*% X0
```
Comment (danielinteractive) [inline R/Model-class.R:2132]: same here

File: R/mcmc.R [hunk 2]
```diff
@@ -396,24 +396,22 @@ setMethod("mcmc",
                      ...){
 
               ## decide whether we sample from the prior or not
-              from_prior <- data@nObs == 0L
+              from_prior <- data@nObs == 0L # TODO shouldn't be no DLT only?
```
Comment (danielinteractive) [inline R/mcmc.R:399]: maybe check with @Winnie-Yeung

**Review by danielinteractive — COMMENTED** [review]
thx Wojciech! looks good so far

File: R/mcmc.R (hunk 2 above)
Comment (wwojciech) [reply R/mcmc.R:399]: Sure, Tx!

Comment (wwojciech) [reply R/mcmc.R]: Winnie does not really remember about how this parameter can be used in this function, so I just removed it.

**Comment by wwojciech** [comment]
Hi @dinakar29 Do you know how we can trigger the checks here? I have tried to do this by using the `Re-run all` button in the actions - but it did not start...
Maybe this is because I first created a draft PR and then marked it as ready for review? But I think that still this should not happen and the checks should be started when I marked this draft PR as ready for review. 
Thank you for your help!

W.

**Review by danielinteractive — APPROVED** [review]

File: inst/WORDLIST [hunk 3]
```diff
@@ -109,6 +109,7 @@ DualSimulationsSummary
 EffFlexi
 Effloglog
 Effmodel
+efficacies
```
Comment (danielinteractive) [inline inst/WORDLIST:112]: is that a real word?... maybe better use `efficacy values`  in text?

**Review by danielinteractive — CHANGES_REQUESTED** [review]

File: inst/WORDLIST (hunk 3 above)
Comment (wwojciech) [reply inst/WORDLIST:112]: I checked in the dictionary and `efficacies` is ok but I can change it to `efficacy values`.

**Review by danielinteractive — APPROVED** [review]

**Pull request merged**